_data/cryptocurrencies.yml

```
@@ -420,6 +420,14 @@ websites:
       - totp
     doc: https://www.luno.com/help/en/articles/1000203420
 
+  - name: MaiCoin
+    url: https://www.maicoin.com
+    img: maicoin.png
+    tfa:
+      - sms
+      - totp
+    doc: https://maicoin2.freshdesk.com/en/support/solutions/articles/32000016116
+
```

Let's add a region code here as well.
```suggestion
    regions:
        - tw
```